Core/Core/Models/Base.cs

```diff
@@ -299,4 +286,35 @@ public Base ShallowCopy()
 
     return myDuplicate;
   }
+
+  #region Obsolete
+  /// <inheritdoc cref="GetId(bool)"/>
+  [Obsolete("Serializer v1 is deprecated, use other overload(s)")]
```

Comment:
Could do, `error = true` implies we've broken it,
 which, in fairness, we may have already inadvertently since this the v1 code hasn't been tested in over a year.

What do you think? I'm all up for a slightly more aggressive series of deprecation with compiler errors.

Reply:
Lets do it... if anybody was using the SerialiserV1 they should not ignore this any longer